src/screens/login-screen.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import {KeyboardAvoidingView, View, StyleSheet} from 'react-native';
+import LoginButton from '../components/buttons/login-button';
+import FormTextInput from '../components/forms/form-text-input';
+
+class LoginScreen extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      passowd: '',
+    };
+  }
+
+  onChangeEmail = email => {
+    this.setState({email});
+  };
+
+  onChangePassword = password => {
+    this.setState({password});
+  };
+
+  render() {
+    return (
+      <KeyboardAvoidingView>
+        <View styles={styles.container}>
```

Comment:
you don't have to wrap view inside keyboard avoiding view, use just simple KeyboardAvoidingView with styles.